packages/now-client/src/utils/index.ts

```diff
@@ -17,17 +17,16 @@ export const API_DELETE_DEPLOYMENTS_LEGACY = '/v2/now/deployments';
 export const EVENTS = new Set([
   // File events
   'hashes-calculated',
-  'file_count',
+  'file-count',
```

Comment:
Is this the reason for the major bump? It should probably be mentioned in the description of the PR so consumers are aware of this change.

Reply:
Not only, I've added it the the PR description, good point 👍